examples/bri-3/src/bri/communication/models/bpiMessageType.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum BpiMessageType {
+  MessageType1,
```

Comment:
Can we please add a TODO to have better descriptive names?

Reply:
fixed